code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -350,7 +350,6 @@
 	ui = SStgui.try_update_ui(user, src, ui)
 	if(!ui)
 		ui = new(user, src, "SmartVend", name)
-		ui.set_autoupdate(FALSE)
```

Comment:
i prefer this stay off since smartfridges basically bottomless

any ui act that returns TRUE will update the ui

you can also manually update it with ui.send_update()